ros_message/src/time.rs

```diff
@@ -97,21 +97,9 @@ impl Time {
     }
 }
 
-fn display_nanos(nanos: &str, f: &mut Formatter<'_>) -> fmt::Result {
-    let split_point = nanos.len() - 9;
-    let characters = nanos.chars();
-    let (left, right) = characters.as_str().split_at(split_point);
-    let right = right.trim_end_matches('0');
```

Comment:
Wouldn't just removing this line do the same trick?

My concern are perfectly valid but strange payloads, like `Duration { sec: -1, nsec: 1 }` and `Duration { sec: -1, nsec: -1 }` printing the same output in your modification of the code.

Reply:
Of course, the if statement below is unnecessary then, as we're always triggering the else case.